specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2378,16 +2586,16 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/agreements": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/assemblies/{assemblyArtifactName}/listContentCallbackUrl": {
+      "post": {
         "tags": [
-          "IntegrationAccountAgreements"
+          "IntegrationAccountAssemblies"
         ],
-        "operationId": "Agreements_ListByIntegrationAccounts",
-        "description": "Gets a list of integration account agreements.",
+        "operationId": "IntegrationAccountAssemblies_ListContentCallbackUrl",
+        "description": "List the content callback url for an integration account assembly.",
```

Comment:
Wondering if the description should change here - as this isn't really a list operation.

Reply:
I agree with you, the underlying endpoint is named not super great, since the endpoint only returns a single callback url. Are you fine with "Get the content callback url for an integration account assembly."? This pattern is used in a bunch of different endpoints in their descriptions and I'll change them all to match

Reply:
Yep - that that sounds good :)